<?xml version="1.0" encoding="UTF-8"?>
<robot generator="Robot 6.0.2 (Python 3.11.2 on win32)" generated="20230503 17:24:55.958" rpa="false" schemaversion="3">
<suite id="s1" name="Mian" source="C:\Users\aflores\Desktop\Robot FrameWork\FInal-Excercise\tests\mian.robot">
<test id="s1-t1" name="Add and Remove element" line="16">
<kw name="Begin Web Test" type="SETUP">
<msg timestamp="20230503 17:24:56.246" level="FAIL">No keyword with name 'Begin Web Test' found.</msg>
<status status="FAIL" starttime="20230503 17:24:56.246" endtime="20230503 17:24:56.246"/>
</kw>
<kw name="End Web Test" type="TEARDOWN">
<msg timestamp="20230503 17:24:56.249" level="FAIL">No keyword with name 'End Web Test' found.</msg>
<status status="FAIL" starttime="20230503 17:24:56.249" endtime="20230503 17:24:56.249">No keyword with name 'End Web Test' found.</status>
</kw>
<status status="FAIL" starttime="20230503 17:24:56.244" endtime="20230503 17:24:56.249">Setup failed:
No keyword with name 'Begin Web Test' found.

Also teardown failed:
No keyword with name 'End Web Test' found.</status>
</test>
<doc>This is a final excercise in the Robot Fram</doc>
<status status="FAIL" starttime="20230503 17:24:55.960" endtime="20230503 17:24:56.251"/>
</suite>
<statistics>
<total>
<stat pass="0" fail="1" skip="0">All Tests</stat>
</total>
<tag>
</tag>
<suite>
<stat pass="0" fail="1" skip="0" id="s1" name="Mian">Mian</stat>
</suite>
</statistics>
<errors>
<msg timestamp="20230503 17:24:56.240" level="ERROR">Error in file 'C:\Users\aflores\Desktop\Robot FrameWork\FInal-Excercise\tests\mian.robot' on line 4: Error in file 'C:\Users\aflores\Desktop\Robot FrameWork\FInal-Excercise\Resources\Common.robot' on line 7: Resource file with 'Test Cases' section is invalid.</msg>
<msg timestamp="20230503 17:24:56.242" level="ERROR">Error in file 'C:\Users\aflores\Desktop\Robot FrameWork\FInal-Excercise\tests\mian.robot' on line 5: Error in file 'C:\Users\aflores\Desktop\Robot FrameWork\FInal-Excercise\Resources\PO\Add_Remove_Elements.robot' on line 7: Resource file with 'Test Cases' section is invalid.</msg>
</errors>
</robot>
